src/main/java/ro/developmentfactory/thedrones/entity/Drone.java

```diff
@@ -0,0 +1,37 @@
+package ro.developmentfactory.thedrones.entity;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.sql.Timestamp;
+import java.util.List;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+public class Drone {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id_drone")
+    private Long idDrone;
+
+    @Column(name = "name")
+    private String name;
+    @Column(name = "count_move")
+    private int countMove;
+    @Column(name = "created_at")
+    private Timestamp createdAt;
```

Comment:
and please use `OffsetDateTime` every time you need to use timestamps in the code, you can also use it here. this way you have much more control over the timestamps and no bugs caused by implementation

Reply:
done